h2o-core/src/main/java/hex/grid/GridSearch.java

```diff
@@ -67,23 +67,28 @@
  * }</pre>
  *
  * @see HyperSpaceWalker
- * @see #startGridSearch(Key, HyperSpaceWalker, int)
+ * @see #startGridSearch(Key, HyperSpaceWalker, boolean, int)
  */
 public final class GridSearch<MP extends Model.Parameters> extends Keyed<GridSearch> {
   public final Key<Grid> _result;
   public final Job<Grid> _job;
+  public final boolean _exportFrames;
   public final int _parallelism;
 
   /** Walks hyper space and for each point produces model parameters. It is
    *  used only locally to fire new model builders.  */
   private final transient HyperSpaceWalker<MP, ?> _hyperSpaceWalker;
 
-  private GridSearch(Key<Grid> gkey, HyperSpaceWalker<MP, ?> hyperSpaceWalker, final int parallelism) {
+  private GridSearch(
+      Key<Grid> gkey, HyperSpaceWalker<MP, ?> hyperSpaceWalker, 
+      final boolean exportFrames, final int parallelism
+  ) {
     assert hyperSpaceWalker != null : "Grid search needs to know how to walk around hyper space!";
     _hyperSpaceWalker = hyperSpaceWalker;
     _result = gkey;
     String algoName = hyperSpaceWalker.getParams().algoName();
     _job = new Job<>(gkey, Grid.class.getName(), algoName + " Grid Search");
+    _exportFrames = exportFrames;
```

Comment:
instead of passing this parameter, I think it makes more logical sense to pass the recovery object - where the grid can store recovery information; if this parameter is not passed - nothing happens